scripts/linters/general_purpose_linter.py

```diff
@@ -363,7 +363,11 @@ def _check_for_mandatory_pattern_in_file(
         # found in the file.
         pattern_found_list = []
         error_messages = []
-        file_content = self.file_cache.readlines(filepath)
+
+        try:
+            file_content = self.file_cache.readlines(filepath)
+        except Exception as e:
+            raise Exception('%s %s' % (filepath, e))
```

Comment:
Do we still need this change?

Reply:
Yes, I couldn't run pylint locally because it was catching all the .direnv files. So this change actually helps show the filepath it is failing on.